c2corg_common/fields_outing.py

```diff
@@ -69,9 +69,14 @@
             'glacier_rating',
             'avalanche_signs',
             'locales.avalanches',
+            'ski_rating',
+            'labande_ski_rating'
         ],
         'required': DEFAULT_REQUIRED,
-        'listing': DEFAULT_LISTING
+        'listing': DEFAULT_LISTING + [
+            'ski_rating',
+            'labande_ski_rating'
+            ]
```

Comment:
This indentation is probably not consistent => align with ``'listing'``. Ditto with the other changes below.